website/docs/docs/get-started/develop-in-the-cloud.md

```diff
@@ -2,36 +2,68 @@
 title: "Develop in the Cloud"
 id: develop-in-the-cloud
 description: "Develop, test, run, and build in the Cloud IDE."
-sidebar_label: "Develop in the Cloud"
+sidebar_label: Develop in the Cloud
 ---
 
+:::info 📌
+
+The new and refreshed Cloud IDE will be open to General Availability by end of October 2022. The new Cloud IDE and includes performance upgrades, ergonomics improvements, and some delightful enhancements!
+
+Review the [new Cloud IDE features](/docs/get-started/dbt-cloud-features#ide-features) and [our release blog](www.getdbt.com/blog/new-improved-cloud-ide) to learn more.
```

Comment:
www.getdbt.com/blog/new-improved-cloud-ide is a broken link. maybe it'll go live tomorrow?